pytorch_lightning/core/lightning.py

```diff
@@ -1493,6 +1493,42 @@ def configure_gradient_clipping(self, optimizer, optimizer_idx, gradient_clip_va
             optimizer, gradient_clip_val=gradient_clip_val, gradient_clip_algorithm=gradient_clip_algorithm
         )
 
+    def lr_scheduler_step(
+        self,
+        scheduler: Any,
+        optimizer_idx: Optional[int] = None,
+        monitor_val: Optional[Union[float, torch.Tensor]] = None,
```

Comment:
Let's call this metrics as it is the expected API from PyTorch.

Reply:
Should we provide the epoch and the current optimizer step count ?

Reply:
we can, but it's directly accessible using `self`.
also Carlos comment here: https://github.com/PyTorchLightning/pytorch-lightning/pull/10249#discussion_r741031873